src/wazuh_modules/vulnerability_detector/wm_vuln_detector.c

```diff
@@ -2020,6 +2023,9 @@ int wm_vuldet_report_agent_vulnerabilities(sqlite3 *db, agent_software *agent, w
         goto end;
     }
 
+    //Clean the vulnerabilities in the agent database
+    wdb_agents_vuln_cve_clear(atoi(agent->agent_id), NULL);
```

Comment:
We can re-use the socket from **wm_vuldet_get_wdb_socket()**

Reply:
Fixed!